internal/pkgecosystem/ecosystem.go

```diff
@@ -5,20 +5,28 @@ import (
 	"strings"
 )
 
-// RunPhase
-// Represents a way to 'run' a package during its usage lifecycle
-// This is relevant to dynamic analysis
+// Ecosystem represents an open source package ecosystem from which packages can be downloaded
+type Ecosystem string
+
+// RunPhase represents a way to 'run' a package during its usage lifecycle.
+// This is relevant to dynamic analysis.
 type RunPhase string
 
 const (
 	Import  RunPhase = "import"
 	Install RunPhase = "install"
+
+	CratesIO  Ecosystem = "crates.io"
+	NPM       Ecosystem = "npm"
+	Packagist Ecosystem = "packagist"
+	PyPi      Ecosystem = "pypi"
+	Rubygems  Ecosystem = "rubygems"
```

Comment:
nit: RubyGems is the right capitalisation.

Reply:
Will fix in next PR